pkgs/os-specific/linux/pam_gnupg/default.nix

```diff
@@ -0,0 +1,24 @@
+{stdenv, fetchFromGitHub, autoconf, automake, libtool, pam, gnupg }:
+
+with stdenv.lib;
+
+stdenv.mkDerivation rec {
+  pname = "pam_gnupg";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "cruegge";
+    repo = "pam-gnupg";
+    rev = "fbd75b720877e4cf94e852ce7e2b811feb330bb5";
```

Comment:
Why not use https://github.com/cruegge/pam-gnupg/releases/tag/v0.1 ?

Reply:
Good point. I just ended up using a release in the new expression since it contains a security fix.